docs/advanced.rst

```diff
@@ -585,6 +585,106 @@ Discovery
+
+The SIMPLE discovery protocol resolves the establishment of the end-to-end connection between various
+RTPS entities communicating via the RTPS protocol. Fast-RTPS implements the SIMPLE discovery protocol to
+provide compatibility with the `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_.
+The specification splits up the SIMPLE discovery protocol into two independent protocols:
+
+- **Simple Participant Discovery Protocol (SPDP):** specifies how Participants discover each other in the network; it
+  announces and detects the presence of participants in a domain.
+
+- **Simple Endpoint Discovery Protocol (SEDP):** defines the protocol adopted by the discovered participants for the
+  exchange of information in order to discover the RTPS entities contained in each of them, i.e. the writers and
+  readers Endpoints.
+
++--------------------------+----------------------------------------------------------+------------+---------+
+| Name                     | Description                                              | Type       | Default |
++==========================+==========================================================+============+=========+
+| `Initial Announcements`_ | It defines the behavior of the RTPSParticipant           | ``uint32`` | 5       |
+|                          | initial announcements.                                   |            |         |
++--------------------------+----------------------------------------------------------+------------+---------+
+| `Simple EDP Attributes`_ | It defines the use of the SIMPLE protocol as a discovery | ``bool``   | true    |
+|                          | protocol.                                                |            |         |
++--------------------------+----------------------------------------------------------+------------+---------+
+
+
+.. _`Initial Announcements`:
+
+Initial Announcements
+---------------------
+
++---------+--------------------------------------------------------------------+----------------+---------+
+| Name    | Description                                                        | Type           | Default |
++=========+====================================================================+================+=========+
+| count   | It defines the number of announcements to send at start-up.        | ``uint32``     | 5       |
++---------+--------------------------------------------------------------------+----------------+---------+
+| period  | It defines the specific period for initial announcements.          | ``Duration_t`` | 100ms   |
++---------+--------------------------------------------------------------------+----------------+---------+
+
++-----------------------------------------------------------------+
+| **C++**                                                         |
++-----------------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp                      |
```

Comment:
The snippet has `use_PublicationWriterANDSubscriptionReader` two times. One of them should be the other one